collect_app/src/main/java/org/odk/collect/android/utilities/ToastUtils.java

```diff
@@ -1,37 +1,61 @@
 package org.odk.collect.android.utilities;
 
+import android.view.Gravity;
 import android.widget.Toast;
 
 import org.odk.collect.android.application.Collect;
 
-
 public class ToastUtils {
 
     private ToastUtils() {
 
     }
 
     public static void showShortToast(String message) {
-        showToast(message, Toast.LENGTH_SHORT);
+        showToast(message, Toast.LENGTH_SHORT, false);
     }
 
     public static void showShortToast(int messageResource) {
-        showToast(messageResource, Toast.LENGTH_SHORT);
+        showToast(messageResource, Toast.LENGTH_SHORT, false);
     }
 
     public static void showLongToast(String message) {
-        showToast(message, Toast.LENGTH_LONG);
+        showToast(message, Toast.LENGTH_LONG, false);
     }
 
     public static void showLongToast(int messageResource) {
-        showToast(messageResource, Toast.LENGTH_LONG);
+        showToast(messageResource, Toast.LENGTH_LONG, false);
+    }
+
+    public static void showShortToastCentered(String message) {
+        showToast(message, Toast.LENGTH_SHORT, true);
+    }
+
+    public static void showShortToastCentered(int messageResource) {
+        showToast(messageResource, Toast.LENGTH_SHORT, true);
+    }
+
+    public static void showLongToastCentered(String message) {
+        showToast(message, Toast.LENGTH_LONG, true);
+    }
+
+    public static void showLongToastCentered(int messageResource) {
+        showToast(messageResource, Toast.LENGTH_LONG, true);
     }
 
-    private static void showToast(String message, int duration) {
-        Toast.makeText(Collect.getInstance(), message, duration).show();
+    private static void showToast(String message, int duration, boolean centered) {
+        Toast toast = Toast.makeText(Collect.getInstance(), message, duration);
+        if (centered) {
+            toast.setGravity(Gravity.CENTER, 0, 0);
```

Comment:
Try adding this to modify the font size:
```Java
    ViewGroup group = (ViewGroup) toast.getView();
    TextView message = (TextView) group.getChildAt(0);
    message.setTextSize(**size you want**);
```
And set this a parameter in method `showToast()`, setting a default value is good in case we don’t want to change the text size.

Reply:
Agreed that the text should be larger for the toasts that happen in the form filling context (middle toasts).